README.md

```diff
@@ -1430,7 +1430,7 @@ cache :: Future a b -> Future a b
 
 </details>
 
-Returns a Future which caches the resolution value of the given Future so that
+Returns a Future which caches the resolution value or rejection reason of the given Future so that
```

Comment:
Please correct the wrapping on this line, and any following lines which my get affected, such that each line does not exceed the 79 character limit. Sorry for the nitpick.